src/DDoSX/AclGeoIpClient.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\AclGeoIp;
+use UKFast\SDK\DDoSX\Entities\AclGeoIpMode;
+
+class AclGeoIpClient extends BaseClient
+{
+    const GEO_IP_MAP = [];
```

Comment:
using the `AclGeoIpClient` directly gets me `no Route matched with those values` as you need the base path here too

```suggestion
    /**
     * @inheritDoc
     */
    protected $basePath = 'ddosx/';

    const GEO_IP_MAP = [];
```

Reply:
Really they should all extend the DDosX client as this already has the base path, however I tried to keep them consistant with what was already there. I am going to apply the changes now, nice spot